components/homme/src/theta-l/prim_advance_mod.F90

```diff
@@ -118,10 +118,9 @@ subroutine prim_advance_exp(elem, deriv, hvcoord, hybrid,dt, tl,  nets, nete, co
 !                 optimal: for windspeeds ~120m/s,gravity: 340m/2
 !                 run with qsplit=1
 !                 (K&G 2nd order method has CFL=4. tiny CFL improvement not worth 2nd order)
-!   tstep_type=6  KG4+BE/CN  2nd order
 !   tstep_type=7  IMKG254a
 !   tstep_type=8  KG3+BE/CN  KG3 2nd order explicit, 1st order off-centering implicit
-!   tstep_type=9  KG5+BE/CN  KG5(3rd order linear, 3.87CFL) + BE/CN,  2nd order IMEX
+!   tstep_type=9  KGU53+BE/CN  KGU53 3rd order explicit, 1st order off-centering implicit
```

Comment:
Is this still supposed to say "1st order off-centering implicit"?

Reply:
fixed